csharp/extractor/Semmle.Extraction/Context.cs

```diff
@@ -41,42 +41,12 @@ public SemanticModel GetModel(SyntaxNode node)
         /// </summary>
         public readonly TrapWriter TrapWriter;
 
-        int GetNewId() => TrapWriter.IdCounter++;
-
-        /// <summary>
-        /// Creates a new entity using the factory.
-        /// </summary>
-        /// <param name="factory">The entity factory.</param>
-        /// <param name="init">The initializer for the entity.</param>
-        /// <returns>The new/existing entity.</returns>
-        public Entity CreateEntity<Type, Entity>(ICachedEntityFactory<Type, Entity> factory, Type init) where Entity : ICachedEntity where Type : struct
-        {
-            return CreateNonNullEntity(factory, init);
-        }
-
         /// <summary>
-        /// Creates a new entity using the factory.
+        /// Holds if assembly information should be prefixed to TRAP labels.
         /// </summary>
-        /// <param name="factory">The entity factory.</param>
-        /// <param name="init">The initializer for the entity.</param>
-        /// <returns>The new/existing entity.</returns>
-        public Entity CreateNullableEntity<Type, Entity>(ICachedEntityFactory<Type, Entity> factory, Type init) where Entity : ICachedEntity
-        {
-            return init == null ? CreateEntity2(factory, init) : CreateNonNullEntity(factory, init);
-        }
+        public readonly bool AddAssemblyTrapPrefix;
```

Comment:
I tend to use `Should` and `Is` as prefixes for these `bool` members, such as `IsValid`, `ShouldAddAssemblyTrapPrefix`. But I've seen that this is not always the case in the codebase.